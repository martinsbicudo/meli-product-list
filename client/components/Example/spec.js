import React from 'react';
import Example from '.';

describe('Meli - Components - <Example />', () => {
  const createWrapper = () => mount(<Example />);

  it('Wrapper Snapshot', () => {
    const Wrapper = createWrapper();
    expect(Wrapper).toMatchSnapshot();
  });
});
