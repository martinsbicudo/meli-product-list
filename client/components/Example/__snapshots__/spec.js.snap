// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meli - Components - <Example /> Wrapper Snapshot 1`] = `
<Example>
  <p>
    Example
  </p>
</Example>
`;
